pkg/daemon/ceph/client/filesystem_mirror.go

```diff
@@ -79,7 +79,12 @@ func DisableFilesystemSnapshotMirror(context *clusterd.Context, clusterInfo *Clu
 	// Run command
 	output, err := cmd.Run()
 	if err != nil {
-		return errors.Wrapf(err, "failed to disable ceph filesystem snapshot mirror for filesystem %q. %s", filesystem, output)
+		// We cannot wrap here since errors.Wrapf will return *errors.withStack which is not exported so we can't assert it
+		// The call reads *exec.ExitError so won't be work
+		if code, err := exec.ExtractExitCode(err); err == nil && code != int(syscall.ENOTSUP) {
+			logger.Errorf("failed to disable ceph filesystem snapshot mirror for filesystem %q. %s. %v", filesystem, output, err)
```

Comment:
Why only log the error and not return it?

Reply:
// We cannot wrap here since errors.Wrapf will return *errors.withStack which is not exported so we can't assert it
		// The call reads *exec.ExitError so won't be work